src/mixin/java/com/gtnewhorizons/angelica/mixins/early/sodium/MixinTileEntityMobSpawner.java

```diff
@@ -0,0 +1,15 @@
+package com.gtnewhorizons.angelica.mixins.early.sodium;
+
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.tileentity.TileEntityMobSpawner;
+import org.spongepowered.asm.mixin.Mixin;
+
+@Mixin(TileEntityMobSpawner.class)
+public class MixinTileEntityMobSpawner extends TileEntity {
+
+    @Override
+    public double getMaxRenderDistanceSquared() {
+        return 256d; // 16 blocks
```

Comment:
This should be a config similar to the item render distance.  The default is fine

Reply:
The default isn't fine at all, when you enter a rogue like dungeon your fps drops to 25 because it's rendering all the mob spawners but none of them is in direct line of sight anyways.

I can add a setting but the default is going to be 16 blocks not 64.

Reply:
16 blocks is also the threshold of activation for the spawners, so it kind of make sense

Reply:
The default you added.....  Still needs a config

Reply:
Cut the attitude also